# Rover Service
# Turn on/off test led
bool state
---
int8 result